ts-defs/index.d.ts

```diff
@@ -723,6 +730,13 @@ interface ActionOptions {
     speed?: number;
 }
 
+interface TakeElementScreenshotOptions extends ActionOptions {
+    crop?: CropOptions;
+    includeMargins?: boolean;
```

Comment:
update this please with @VasilyStrelyaev 
1) add new options like `includePaddings`
2) add descriptions for `crop`, `include...`, `